integritybackend/actions.py

```diff
@@ -4,11 +4,12 @@
 from .file_util import FileUtil
 from .filecoin import Filecoin
 from .iscn import Iscn
+from .log_helper import LogHelper
```

Comment:
From now on we use LogHelper. It ensures the correct configurations of the logging is applied.